packages/libwasmer/libwasmer.3.0.0~beta/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+
+authors: ["wasmer"]
+maintainer: "engineering@wasmer.io"
+license: "MIT"
+homepage: "https://wasmer.io/"
+
+doc: "https://docs.wasmer.io/"
+bug-reports: "https://github.com/wasmerio/wasmer-ocaml/issues"
+dev-repo: "git+https://github.com/wasmerio/wasmer-ocaml.git"
+tags: ["org:wasmer"]
+
+synopsis: "The official Wasmer library"
+description: """
+This package is used by the wasmer package to trigger the installation
+of the Wasmer library."""
+
+depends: ["conf-rust"]
+
+url {
+	src: "https://github.com/wasmerio/wasmer/archive/refs/tags/3.0.0-beta.tar.gz"
+	checksum: [
+		"md5=0741bb953e47349954b731c4da1386d4"
+		"sha512=38ec15c601c67468bca255a5a757afaedd6e1924662e3d108e63380a06f26e9f73f1aa0ef0700547885b52205a7d3f84c25bf266704b292b84585103c8369212"
+	]
+}
+extra-source "vendor.tar.gz" {
+	src: "https://github.com/wasmerio/wasmer-ocaml/releases/download/1.0.0%2Bapi-fix/vendor.tar.gz"
+	checksum: [
+		"md5=fd33f0fe59ce78d636b1a4178e4ef2f7"
+		"sha512=b0bee5168db59a7b629c4090cbd63adddf29931013b47e46050b1c30fc40937c886cf805eeacf43ab8d83d36204c5fb5abf17e90e178a09ebd224af598bfa9b2"
+	]
+}
+build: [
+	["mkdir" ".cargo"]
+	["mv" "cargo-config" ".cargo/config.toml"]
+	["tar" "xf" "vendor.tar.gz"]
+	["make" "build-capi"]
+	["make" "package-capi"]
```

Comment:
```suggestion
	[make "build-capi"]
	[make "package-capi"]
```

Reply:
"make" vs. `%{make}%` where the former is always `make` (whatever the platform) and the latter is always GNU Make (e.g. `gmake` on BSDs)

Reply:
Fixed.